README.md

```diff
@@ -26,8 +26,9 @@ Usage: svgdigitizer [OPTIONS] COMMAND [ARGS]...
 Options:
   --help  Show this message and exit.
 Commands:
-  cv        Digitize a cylic voltammogram.
+  cv        Digitize a cylic voltammogram and create a frictionless...
   digitize  Digitize a plot.
+  figure    Digitize a figure with units on the axis and create a...

```

Comment:
Why the `...`?

Reply:
Same as below. `byexample` creates a cut-off at 80 characters.

Reply:
see #187